extensions/wikia/AdEngine/js/provider/turtle.js

```diff
@@ -25,31 +25,27 @@ define('ext.wikia.adEngine.provider.turtle', [
 		return ret;
 	}
 
-	function fillInSlot(slotName, slotElement, success, hop) {
-		log(['fillInSlot', slotName, slotElement, success, hop], 'debug', logGroup);
+	function fillInSlot(slot) {
+		log(['fillInSlot', slot.name], 'debug', logGroup);
 
+		slot.pre('success', function () {
+			var slotName = slot.name;
```

Comment:
What are the reasons you decided to assign `slot.name` first to the a variable and re-used it later on?